### DISCUSSION TITLE
ContainerRequestContextProvider object

### DISCUSSION THREAD

**Discussion by pri00707** [Netflix/dgs-framework#1963] [Q&A]
How do we get ContainerRequestContextProvider object inside @DgsComponent?

**Comment by paulbakker** [comment]
Do you mean `[ContainerRequestContextProvider](https://jakarta.ee/specifications/platform/8/apidocs/javax/ws/rs/container/containerrequestcontext)` from JAX-RS? That's not used at all for DGS, or Spring Boot in general, so that's not available.
